### ISSUE TITLE
Flickering screens in multiplayer

### ISSUE THREAD

**Issue by knackrack615** [Delta2Force/MCVmComputers#121]
For some reason when playing in multiplayer with another person who also has a computer running, my screen flickers but i can see their screen just fine, and their screen is flickering but they can see my screen just fine.

It only happens if both computers are running at the same time, if any of us turn off their PC the flickering completely disappears.

**Comment by Busaruba2011** [comment]
Is it lan or an actual server?

**Comment by knackrack615** [comment]
It happens on both.

**Comment by Busaruba2011** [comment]
> It happens on both.

What do you mean? Servers and lan?

**Comment by knackrack615** [comment]
Yes, it happens on games "Opened to LAN" and on an actual Fabric servers.

Both the clients and the fabric server only have Fabric API and VMComputers installed (Both latest 1.15.2 versions).

**Comment by Busaruba2011** [comment]
> Yes, it happens on games "Opened to LAN" and on an actual Fabric server.
> 
> Both the clients and the fabric server only have Fabric API and VMComputers installed (Both latest 1.15.2 versions).

Try reinstalling MCVM

**Comment by knackrack615** [comment]
We have completely reinstalled the clients & the server and deleted the VBox VMs and the issue persists.

I have recorded a video demonstrating the issue: https://streamable.com/lbxhmy

**Comment by Busaruba2011** [comment]
> We have completely reinstalled the clients & the server and deleted the VBox VMs and the issue persists.
> 
> I have recorded a video demonstrating the issue: https://streamable.com/lbxhmy
Are you both on the same version of VirtualBox?

**Comment by Busaruba2011** [comment]
@Delta2Force you need to read this

**Comment by knackrack615** [comment]
> > We have completely reinstalled the clients & the server and deleted the VBox VMs and the issue persists.
> > I have recorded a video demonstrating the issue: https://streamable.com/lbxhmy
> 
> Are you both on the same version of VirtualBox?
> Have you got any other mods installed on the server? That might affect it.

Yes, we are using the latest VBox with the latest extension pack (VBox 6.1.16 r140961 & VboxExt 6.1.16)

Like i said in my previous message, this is a fresh install for both the clients and the sever, we have tried reinstalling about 3 times and theres nothing else running besides VMComputers and Fabric API (I have Optifine on my client only so i can zoom, but i have tried without it, it doesn't make a difference).

**Comment by Busaruba2011** [comment]
> > > We have completely reinstalled the clients & the server and deleted the VBox VMs and the issue persists.
> > > I have recorded a video demonstrating the issue: https://streamable.com/lbxhmy
> > 
> > 
> > Are you both on the same version of VirtualBox?
> > Have you got any other mods installed on the server? That might affect it.
> 
> Yes, we are using the latest VBox with the latest extension pack (VBox 6.1.16 r140961 & VboxExt 6.1.16)
> 
> Like i said in my previous message, this is a fresh install for both the clients and the sever, we have tried reinstalling about 3 times and theres nothing else running besides VMComputers and Fabric API (I have Optifine on my client only so i can zoom, but i have tried without it, it doesn't make a difference).

Have you tried without the ext pack? If not, do it.

**Comment by knackrack615** [comment]
Just did, it doesn't make a difference as shown in this video: https://streamable.com/de5ntz

We also came across another weird bug when if there are alot of active pixels on the screen, the screen will stop being streamed to the other player till those pixels clear (In this demo we're using 640x480, 256 colors, it happens with any OS we've tried so far): https://streamable.com/r9k12f

If the pixel bug happens too quickly (eg. a screensaver is playing), the user who's sharing the virtual machine will be kicked from the server with the following message: https://streamable.com/44o9fc

It also looks like for whatever reason the left click mouse button will right click sometimes in the windows VMs we've tried (this can be seen in the latest video)

**Comment by Busaruba2011** [comment]
As I can't develop Mcvmcomputers, you'd have to show @Delta2Force
Sorry I couldn't really help!

**Comment by Busaruba2011** [comment]
Ask @Delta2Force about it
